# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/bugs/reftest.list
# Commit: b7f88e5c537b
# Full Hash: b7f88e5c537bb7d64ba9d4cb58b052650d510638
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers,kvark
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 8e9c1ffdeed8 -r b7f88e5c537b layout/reftests/bugs/reftest.list
--- a/layout/reftests/bugs/reftest.list	Mon Feb 07 22:02:35 2022 +0000
+++ b/layout/reftests/bugs/reftest.list	Mon Feb 07 22:14:47 2022 +0000
@@ -2058,7 +2058,7 @@
 == 1513423-2.html 1513423-2-ref.html
 == 1513423-3.html 1513423-3-ref.html
 pref(layout.accessiblecaret.enabled,true) == 1517385.html 1517385-ref.html
-fuzzy-if(winWidget&&swgl,1-1,12-16) fuzzy-if(cocoaWidget&&swgl,1-1,32-32) fuzzy-if(useDrawSnapshot,2-2,209-209) == 1529992-1.html 1529992-1-ref.html
+fuzzy(0-5,0-2300) == 1529992-1.html 1529992-1-ref.html
 fuzzy-if(Android,9-14,44-60) fails-if(!useDrawSnapshot) == 1529992-2.html 1529992-2-ref.html
 == 1535040-1.html 1535040-1-ref.html
 == 1545360-1.xhtml 1545360-1-ref.xhtml